apps/meteor/ee/server/patches/isUnverifiedContact.ts

```diff
@@ -0,0 +1,48 @@
+import type { AtLeast, ILivechatContact, IOmnichannelRoom } from '@rocket.chat/core-typings';
+import { LivechatContacts } from '@rocket.chat/models';
+
+import { hasLicense } from '../../../app/license/client';
+import { isUnverifiedContact } from '../../../app/livechat/server/lib/Contacts';
+import { settings } from '../../../app/settings/server';
+
+isUnverifiedContact.patch(async (_next, room: AtLeast<IOmnichannelRoom, 'v'>): Promise<boolean> => {
+	const hasContactIdLicense = await hasLicense('contact-id-verification');
+
+	if (hasContactIdLicense) {
+		return false;
+	}
+
+	if (!room.v.contactId) {
+		return false;
```

Comment:
Is it okay to treat old, non-migrated rooms as being verified?
If not we may need to change this to use the visitorId instead, or change the takeInquiry code to also migrate trigger a migration for any room it checks.